openfisca_uk/variables/finance/income.py

```diff
@@ -241,3 +241,58 @@ class base_net_income(Variable):
     entity = Person
     label = "Existing net income for the person to use as a base in microsimulation"
     definition_period = YEAR
+
+
+class is_apprentice(Variable):
+    value_type = bool
+    entity = Person
+    label = u"In an apprenticeship programme"
+    definition_period = YEAR
+
+
+class MinimumWageCategory(Enum):
+    APPRENTICE = "Apprentice"
+    UNDER_18 = "Under 18"
+    BETWEEN_18_20 = "18 to 20"
+    BETWEEN_21_24 = "21 to 24"
+    OVER_24 = "25 or over"
+
+
+class minimum_wage_category(Variable):
+    value_type = Enum
+    possible_values = MinimumWageCategory
+    default_value = MinimumWageCategory.OVER_24
+    entity = Person
+    label = "Minimum wage category"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        return select(
+            [
+                person("is_apprentice", period),
+                age < 18,
+                (age >= 18) & (age <= 20),
```

Comment:
I thought there was a way to use something like `between` for this?

Reply:
True, will use that instead

Reply:
Oh, turns out I need to add `between` to openfisca-tools.